sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1065,24 +1106,37 @@ def append(self, item):
           self._overlay[self._key] = list(self._underlying[self._key])
         self._overlay[self._key].append(item)
 
+    StateType = Union[CopyOnWriteState,
+                      DefaultDict[bytes, CopyOnWriteListProtocol]]
+
     def __init__(self):
+      # type: () -> None
       self._lock = threading.Lock()
-      self._state = collections.defaultdict(list)  # type: DefaultDict[bytes, List[bytes]]
```

Comment:
This is confusing. Why does self._state change its type during the lifetime of StateServicer?
Perhaps it'd be clearer if there were 2 members (_state and _cow_state), or if self._state was initialized to be a CopyOnWriteState in this method? @robertwb

Reply:
This comment isn't a blocker for this PR.